---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

title: Home
layout: default
---

<img style="float: right" src="img/BI-logo-small.png" width="100px"> This page lists free software packages developed by the **Bioinformatics Core** of the [Interdisciplinary Center for Biotechnology Research](http://biotech.ufl.edu) at the [University of Florida](http://ufl.edu).  Software listed here is provided free of charge for academic and research use under the MIT license (see LICENSE file in the individual repositories). 

Most software packages listed in this page were developed for use on HiPerGator, UF's cluster computer, and is therefore installed there. Unless otherwise specificied, they are accessible through the **dibig_tools** module. For information about the packages linked from this page please contact [Alberto Riva](mailto:ariva@ufl.edu). 

The ICBR Bioinformatics Core provides data analysis and software development services for the genomics research community, with a focus on the analysis and interpretation of Next-Gen Sequencing data. For more information about the Bioinformatics core's services please contact [ICBR-Bioinformatics@ad.ufl.edu](mailto:ICBR-Bioinformatics@ad.ufl.edu).

All code available throught this page is Â© University of Florida Research Foundation.

---

# Downloadable software
The following packages are hosted on GitHub.

## [Csvutils](https://github.com/uf-icbr-bioinformatics/csvutils)
A collection of utilities for working with delimited files. Includes a program to automatically convert several delimited files into a single Excel file, and an interactive full-screen viewer for delimited files. This package is available through the [csvutils](https://github.com/uf-icbr-bioinformatics/csvutils) repository

---
## [Bioscripts](https://github.com/uf-icbr-bioinformatics/bioscripts)
This repository contains a collection of stand-alone scripts for various tasks in bioinformatics. They were developed independently for use in various data analysis pipelines. This package is available through the [bioscripts](https://github.com/uf-icbr-bioinformatics/bioscripts) repository.

## [SOI](https://github.com/uf-icbr-bioinformatics/soi)
SOI (Significance Of Intersections) is a command-line program to quickly evaluate whether the intersections among sets (up to four) are significantly high or low. This program is available through the [soi](https://github.com/uf-icbr-bioinformatics/soi) repository.

# Utilities

## [submit](https://github.com/uf-icbr-bioinformatics/submit)
submit.py is a command to submit jobs to clusters running slurm or moabs schedulers. It provides a uniform syntax and several user-friendly improvements over the native submission commands.

# Applications
The following programs are distributed in compiled form only (for GNU/Linux systems). Source code is available upon request.

## [CSCALL](http://compbio.ufl.edu/resources/software/cscall/)
CSCALL is a complete package for the genome-wide analysis of DNA methylation.  At its core is a module to detect single-base conversion events in short reads, specifically designed to analyze the results of sodium bisulfite sequencing experiments to study genome-wide methylation. CSCALL is optimized to work on large genomes and multiple biological replicates, and is able to detect conversion events on a number of different contexts (e.g. CG, GC, CHG, CHH, etc). CSCALL is available from [this page](http://compbio.ufl.edu/resources/software/cscall/).
